pkg/testcase/testcase.go

```diff
@@ -19,6 +22,66 @@ type TestCase struct {
 	IsSubtest bool
 }
 
+type ignoreTestCase struct {
+	Name      string
+	Classname string
+}
+
+var ignoreList = []ignoreTestCase{
+	{Name: "", Classname: "runtime.MemStats"},
+}
+
+// LoadTestSuites loads all reports in provided directory.
+func LoadTestSuites(reportDir string) ([]junit.Suite, error) {
+	testSuites, err := junit.IngestDir(reportDir)
+	if err != nil {
+		return nil, err
+	}
+
+	return getClearedSuites(testSuites), nil
+}
+
+func deleteHelperTest(testElem junit.Test) bool {
+	for _, ignore := range ignoreList {
+		if testElem.Name == ignore.Name && testElem.Classname == ignore.Classname {
+			return true
+		}
+	}
+
+	return false
+}
+
+func addFallbacks(tests []junit.Test) []junit.Test {
```

Comment:
I think it's worth mentioning that this function mutates its input argument.
```suggestion
// Makes sure the passed tests all have class and test names set to a non-empty value, and returns the argument.
func addFallbacks(tests []junit.Test) []junit.Test {
```